rfc6962-bis.xml

```diff
@@ -351,18 +351,30 @@ d0 d1   d2 d3           d0 d1   d2 d3  d4 d5</artwork>
+          </t>
+          <t>
+            Wildcard <spanx style="verb">*</spanx> labels MUST NOT be redacted. However, if the complete leftmost label of a DNS-ID or CN-ID is <spanx style="verb">*</spanx>, it is considered redacted for the purposes of determining if the label to the right may be redacted. For example, if a certificate contains a DNS-ID of <spanx style="verb">*.top.secret.example.com</spanx>, then the corresponding DNS-ID in the precertificate could contain <spanx style="verb">*.REDACTED(top).REDACTED(secret).example.com</spanx> instead, but not <spanx style="verb">REDACTED(*).REDACTED(top).REDACTED(secret).example.com</spanx> instead.
           </t>
         </section>
-        <section title="Redacted Labels Certificate Extension" anchor="redaction_extension">
+        <section title="Redacted SAN Certificate Extension" anchor="redacted_san_extension">
+          <t>
+            The Redacted SAN extension is a non-critical extension (OID 1.3.101.77) identical in structure to the subjectAltName extension, the difference being that dNSName entries MAY contain redacted labels. The purpose of this extension is to inform TLS clients of the dNSNames that were redacted and the degree of redaction, while minimizing the complexity of TBSCertificate reconstruction, as described in <xref target="reconstructing_tbscertificate"/>.
+          </t>
           <t>
-            When a precertificate contains one or more <spanx style="verb">?</spanx> labels, a non-critical extension (OID 1.3.101.77, whose extnValue OCTET STRING contains an ASN.1 SEQUENCE OF INTEGERs) MUST be added to the corresponding certificate. The purpose of this extension is to enable TLS clients to reconstruct the TBSCertificate component of the precertificate from the certificate, as described in <xref target="reconstructing_tbscertificate"/>.
+            The Redacted SAN extension MUST be present in the certificate corresponding to the precertificate that was logged with it.
           </t>
           <t>
-            Each INTEGER MUST have a value of zero or more. The first INTEGER indicates the total number of <spanx style="verb">?</spanx> labels in the precertificate's first DNS-ID; the second INTEGER does the same for the precertificate's second DNS-ID; etc. The last INTEGER does the same for the precertificate's zero or more CN-IDs. There MUST NOT be more INTEGERs than there are DNS-IDs (plus one, if any CN-IDs are present); if there are fewer INTEGERs than this, the shortfall is made up by implicitly repeating the last INTEGER.
+            The purpose behind hashing the redacted labels is to allow the legitimate domain owner to identify if the redacted precertificate correlates to one of the subdomains it's aware of or a new, potentially unautorized, subdomain.
```

Comment:
(1) changed precertificate -> label.
(2) I think the *how* is straightforward (if the reader was able to follow generation of redacted labels, they could figure out how to find out a match between a redacted label and a potential source label)